tests/test_dagmc_file.py

```diff
@@ -77,6 +80,88 @@ def test_set_entityset_ranges():
             type_range == single_cube.entityset_ranges[set_type])
     assert(all(test_pass))
 
+def test_set_dimension_meshset():
+    """
+    Tests different aspects of the set_dimension_meshset function
+    """
+    single_cube = ds.DagmcStats(test_env[1]['input_file'])
+    range = single_cube._my_moab_core.get_entities_by_type(
+                            single_cube.root_set, types.MBENTITYSET)
+    assert(sorted(single_cube.dim_dict.values())==list(range[-4:]))
```

Comment:
dimlist = ['node', 'curves', 'surfaces', 'volumes']
for dim, meshset in dim_dict.values():
    if dim in dimlist:
        True
    if meshset in rangs[-4:]:
        True

Reply:
`len(set(dim_dict.values())) == 4`
`len(set(dim_dict.keys())) == 4`

Reply:
`sorted(dim_dict.keys()) == sorted(dim_list)`
`sorted(dim_dict.values()) == sorted(range[-4:])`

Reply:
lots of comments- the last comment I think is the most succinct way to test the keys and values